claro.theme

```diff
@@ -99,12 +99,21 @@ function claro_preprocess_block(&$variables) {
 }
 
 /**
- * Implements hook_preprocess_HOOK() for block admin page templates.
+ * Implements template_preprocess_HOOK() for admin_block.
+ */
+function claro_preprocess_admin_block(&$variables) {
+  if (!empty($variables['block']['content'])) {
+    $variables['block']['content']['#attributes']['class'][] = 'admin-list--panel';
```

Comment:
Maybe there's something wrong with the condition but this class doesn't ever get added as far as I can tell.

Reply:
I have installed it on`umami_demo` install profile. I checked all different menus on admin UI and none of them had this class added. I'm not sure if this is the right way to test is 🙂

Reply:
I used a wrong hook (see previous comment).